README.md

```diff
@@ -62,11 +62,32 @@ There is only one modifiable property in settings:
    //    - "on"
    //    - "off"
    //    - "smart": Automatic formatting is only enabled if there is a `black` section in the project's `pyproject.toml`
-   "format_on_save": "on"
+   "format_on_save": "on",
+   // Black [OPTIONS]
+   "line_length": "88",
+   "skip_source_first_line": false,
+   "skip_string_normalization": false,
+   "skip_magic_trailing_comma": false,
+   "experimental_string_processing": false
```

Comment:
The configuration fields of black are more than these fields. If you want to use sublime-settings as a configuration file, you should put these fields in settings so that users can configure them correctly:

```js
{
   "format_on_save": "on",
   "settings": {
       "target_versions": [],
       "line_length": 88,
       //...
       "python_cell_magics": []
   }
}
```

For all fields, please refer to: https://github.com/psf/black/blob/main/src/black/mode.py#L170-L178

Reply:
Sure, I'll expose all Mode kwargs as black options.